Nataly_Bondarenko/HW_12/Photo.py

```diff
@@ -0,0 +1,102 @@
+from Iphoto_shots import IPhoto_shots
+from Ipost_processing import IPost_processing
+
+
+# inheritance
+class PhotoCenter(IPhoto_shots, IPost_processing):
+    def __init__(self, number_of_clients: int):
+        self.__sorted = False
+        self.__camera = False
+        if number_of_clients > 0:
+            self.__number_of_clients = number_of_clients
+        else:
+            raise ValueError("Number of the clients can not be less than 1")
+        self.__photo_session_price = 150
+        self.__number_of_photos = 200
+        self.__discount = 0, 1
+
+    @property
+    def number_of_clients(self):
+        return self.__number_of_clients
+
+    @number_of_clients.setter
+    def number_of_clients(self, new_number_of_clients: int):
+        if new_number_of_clients < 1:
+            print("The number of participants in the photo session cannot be less than 1")
+        elif 2 >= new_number_of_clients >= 1:
+            print(f'From 1 to 2 people this is individual or couple photo session. '
+                  f'The total price for {self.__number_of_photos} will be {self.__photo_session_price}')
+        else:
+            print(f'From 2 and more people this is a family photo session. '
+                  f'The total price for {self.__number_of_photos} will be {self.__photo_session_price * 2}')
+
+    # encapsulation
+    def preparing(self):
+        self.meet_customers()
+        self.turn_on_the_camera()
+
+    # encapsulation
+    def photo_session(self):
+        self.make_shots()
+        self.turn_off_the_camera()
+        self.say_goodbye_to_customers()
+
+    # encapsulation
```

Comment:
Encapsulation - detecting access to the constituent methods and variables. Are you sure that you write correct comment?